.uncrustify.cfg

```diff
@@ -10,6 +10,7 @@ indent_func_def_param           = true
 indent_func_proto_param         = true
 indent_func_param_double        = true
 indent_switch_case              = 8
+nl_fdef_brace                   = add
```

Comment:
with `nl_fdef_brace = add` I get

    22 files changed, 438 insertions(+), 219 deletions(-)

while with `nl_fdef_brace = remove` I get

    21 files changed, 165 insertions(+), 327 deletions(-)

which seems less invasive. Thus we explicitly change coding style?